test/test_cpp20/ranges.cpp

```diff
@@ -0,0 +1,48 @@
+#include "pch.h"
+#include <algorithm>
+#include <ranges>
+
+TEST_CASE("ranges")
+{
+	{
```

Comment:
4-space tabs please.

Reply:
My bad! That's something an `.editorconfig` or `.clang-format` file could help avoid ;)

Resolved